### PR TITLE
Light signup origami

### PR DESCRIPTION
**Pull request by tavvy**
Now that light sign up is available as an origami component its
template can live as an html file inside views. This means the
logic can be simplified and offloaded to the handlebars render.

### CODE REVIEW

File: views/partials/light-signup.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="o-email-only-signup o-email-only-signup__visually-hidden" data-o-component="o-email-only-signup" data-trackable="light-signup" aria-hidden="true">
+	<div class="o-email-only-signup__inner">
+		<div class="o-email-only-signup__close" data-o-email-only-signup-close></div>
+		<h2 class="o-email-only-signup__heading">Get a taste of the FT</h2>
```
Comment (i-like-robots) [inline views/partials/light-signup.html]: h3

File: views/partials/light-signup.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+		<p class="o-email-only-signup__tagline">Sample the FT, with the top stories sent FREE to your inbox every morning.</p>
+		<p class="o-email-only-signup__text">For 1 week, you’ll get an email with free access to the 3 most read FT stories.</p>
+		<div class="o-email-only-signup__secondary" data-o-email-only-signup-completion-message>
+			<form class="o-email-only-signup__form" data-o-email-only-signup-form>
+				<input data-o-grid-colspan="7" type="text" id="email" name="email" class="o-email-only-signup__email o-forms-text" data-trackable="email">
```
Comment (i-like-robots) [inline views/partials/light-signup.html]: label?

**Comment by tavvy** [comment]
👍 to your comments. I will adjust the origami components template html.

File: views/partials/light-signup.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<div class="o-email-only-signup o-email-only-signup__visually-hidden" data-o-component="o-email-only-signup" data-trackable="light-signup" aria-hidden="true">
+	<div class="o-email-only-signup__inner">
+		<div class="o-email-only-signup__close" data-o-email-only-signup-close></div>
```
Comment (i-like-robots) [inline views/partials/light-signup.html]: Presuming this is an X - should be a button element with visually hidden 'close' text I reckon

**Comment by i-like-robots** [comment]
squash and 🚢 !

**Pull request merged**